webapp/src/components/sidebar_right/gitlab_items.tsx

```diff
@@ -126,7 +126,7 @@ function GitlabItems({item, theme}: GitlabItemsProps) {
         </span>
     ) : null;
 
-    const labels: JSX.Element[] | null = item.labels_with_details ? getGitlabLabels(item.labels_with_details) : null;
+    const labels: JSX.Element[] | undefined = item.labels_with_details ? getGitlabLabels(item.labels_with_details) : null;
```

Comment:
Type of the element is `JSX.Element[] | undefined` but we are assigning `null` to it?...... Fix this here and check if the same thing is being repeated somewhere else